spec/Overview-WebCryptoAPI.xml

```diff
@@ -10092,7 +10092,7 @@ required <a href="#dfn-CryptoKey">CryptoKey</a> <dfn id="dfn-EcdhKeyDeriveParams
                         <li>
                           <p>
                             Let <var>secret</var> be the result of applying the field element to
-                            <a href="#dfn-octet-string">octet string</a> conversion defined in Section ? of <a href="#X9.63">X9.63</a>
+                            <a href="#dfn-octet-string">octet string</a> conversion defined in Section 6.2 of <a href="#RFC6090">RFC6090</a>
```

Comment:
I have no idea what is going on here - my version has the full section reference in the text.  I have wrapped the line to see if that fixes what you see.

Note - changing the correct reference would render this entire edit moot as it would be correct at that point.  One then has a matter of the desirability of a freely available spec vs one behind a paywall.